src/main/java/ua/goit/gojava32/kickstarter/dao/ImageDAO.java

```diff
@@ -0,0 +1,6 @@
+package ua.goit.gojava32.kickstarter.dao;
+
+import ua.goit.gojava32.kickstarter.model.Image;
+
+public interface ImageDAO extends GenericCRUDDAO<Image> {
```

Comment:
Also GenericCrudDao and AbstactCrudDao confuse me - as I expect them to do the same